ups/prompt_prototype.table

```diff
@@ -7,10 +7,13 @@ setupRequired(sconsUtils)
 setupRequired(utils)         # Used by scripts in bin.src
 
 # Used by middleware_interface module.
+setupRequired(ap_pipe)
```

Comment:
Worth adding a comment that this line is required because of the hardcoded pipeline references, not from any code.